include/eve/module/core/regular/converter.hpp

```diff
@@ -104,7 +104,7 @@ template<scalar_value T> using converter_type = decorated<convert_to_<T>()>;
 //!   namespace eve
 //!   {
 //!      template< eve::value T >
-//!      as_wide_as<T, float> float32(T x) noexcept; //1
+//!      as_wide_as<T, float> float32(T x) noexcept;
```

Comment:
there is still a //2 so wjy removing the // 1 ?

Reply:
done